### DISCUSSION TITLE
EventEmitter Error

### DISCUSSION THREAD

**Discussion by Ruinedwastaken** [Androz2091/discord-player#625] [Q&A]
(node:445) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 error listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit
(node:445) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 data listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit

how do i fix this and where do i put that emitter.setMaxListeners()?

**Comment by Ruinedwastaken** [comment]
this occurs when i loop a song

i'm using the temporary solution of the 404 error, so maybe the loop has to be different or something

**Comment by twlite** [comment]
This isn't error, it's just a warning. Your code is adding multiple event listeners (for example: `player.on("something")` inside your command)

**Comment by twlite** [comment]
> i'm using the temporary solution of the 404 error, so maybe the loop has to be different or something
🤔 I don't think that has anything to do with this but I will check

**Comment by Ruinedwastaken** [comment]
> 
> 
> This isn't error, it's just a warning. Your code is adding multiple event listeners (for example: `player.on("something")` inside your command)

ok, i'll check

**Comment by Ruinedwastaken** [comment]
oh, they are all events (11 events) so how do i fix it?

**Comment by twlite** [comment]
You dont add multiple listeners for same event
